pypreprocess/time_diff.py

```diff
@@ -172,7 +172,7 @@ def plot_tsdiffs(results, use_same_figure=True):
     T = len(results['volume_means'])
     S = results['slice_mean_diff2'].shape[1]
     mean_means = np.mean(results['volume_means'])
-    scaled_slice_diff = results['slice_mean_diff2'] / mean_means
+    scaled_slice_diff = results['slice_mean_diff2'] / mean_means ** 2
```

Comment:
No, this does not change the result, since the normalizer is a constant (neither volume- nor slice- nor time-specific). I simply think that the final quantity should be dimensionless.